docs/rules/no-deeply-nested-dependent-keys-with-each.md

```diff
@@ -18,11 +18,11 @@ Examples of **correct** code for this rule:
 
 ```js
 displayNames: computed('owners.@each.name', function() {
-  return this.owners.mapBy(owners, 'name');
+  return this.owners.mapBy('owner', 'name');
 }),
 
 owners: computed('todos.@each.owner', function() {
-  return this.todos.mapBy(todo, 'owner');
+  return this.todos.mapBy('todo', 'owner');
```

Comment:
```suggestion
  return this.todos.mapBy('owner');
```
(mapBy)[https://emberjs.com/api/ember/3.7/classes/EmberArray/methods/mapBy?anchor=mapBy] only takes a single parameter

Reply:
Woops! I try to fix an error and end up not fixing it all :)